.circleci/config.yml

```diff
@@ -94,4 +100,10 @@ workflows:
             branches:
               only:
                 - master
-                - /pull.*/
\ No newline at end of file
+                - /pull.*/
+      - release:
+          filters:
+            branches:
+              ignore: /.*/
```

Comment:
It looked unnecessary at glance but then turned out necessary - I had to add this branch ignore pattern to actually avoid `release` jobs being built against untagged commits from any branch.